azure-spring-boot-parent/pom.xml

```diff
@@ -242,27 +260,51 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>cobertura-maven-plugin</artifactId>
-                <version>2.7</version>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.3</version>
+                <executions>
+                    <execution>
+                        <id>prepare-ut-agent</id>
+                        <phase>process-test-classes</phase>
+                        <goals>
+                            <goal>prepare-agent</goal>
+                        </goals>
+                        <configuration>
+                            <excludes>
+                                <exclude>com/microsoft/azure/spring/boot/autoconfigure/common/GetHashMac.class</exclude>
+                                <exclude>com/microsoft/azure/**/Constants.class</exclude>
+                                <exclude>com/microsoft/azure/**/AzureADJwtTokenFilter.class</exclude>
+                                <exclude>com/microsoft/azure/**/Userprincipal.class</exclude>
+                            </excludes>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>report</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>check</goal>
+                            <goal>report</goal>
+                        </goals>
+                        <configuration>
+                            <haltOnFailure>false</haltOnFailure>
```

Comment:
Had to change it to false, because the azure-spring-boot Maven module has only 36% of branch coverage.